### ISSUE TITLE
The value of a disabled DropDownList is not announced by NVDA/JAWS with Chrome

### ISSUE THREAD

**Issue by martintabakov** [telerik/kendo-ui-core#7256]
### Bug report

The value of a disabled DropDownList is not announced in the below screen reader -> browser combinations:

NVDA with Chrome
JAWS with Chrome

The value is correctly announced by NVDA in Firefox.

After testing the scenario with the ComboBox, the value was correctly announced in all screen reader -> browser combinations.

### Reproduction of the problem

1. Open this Dojo example in Chrome with JAWS/NVDA - [https://runner.telerik.io/fullscreen/@Dimitar-Goshev/OSELULAr](https://runner.telerik.io/fullscreen/@Dimitar-Goshev/OSELULAr)
2. Click the Enable checkbox
3. Select a value from the DropDownList
4. Click the Enable checkbox again.
5. Focus the Checkbox and press **the down arrow key.**

### Current behavior

The screen reader announces the disabled state of the DropDownList, but does not announce its value.

**Note** - the value of the regular select element below the DropDownList is announced.

### Expected/desired behavior

The value of the disabled DropDownList should be announced in the above mentioned combinations.

### Environment

- **Kendo UI version:** 2023.3.1114
- **Browser:** Chrome 120
